### ISSUE TITLE
PM-1072

### ISSUE THREAD

**Issue by mayani** [pegasus-isi/pegasus#1186]
PM-1072

**Comment by mayani** [comment]
**Author:** @juve

I think the first part, about skipping the bundled worker package, is caused by having PEGASUS_HOME for my compute site set to an incorrect path, which causes the planner to not add the worker package to the transfer_input_files.

**Comment by mayani** [comment]
**Author:** @vahi

Yes that is right. A PEGASUS_HOME profile value in the site catalog will disable the shipping for worker package with the jobs. 

The reasoning, is that the user has already indicated what PEGASUS version they want to use by specifying a pegasus  installation in the site catalog

**Comment by mayani** [comment]
**Author:** @juve

I changed two things in pegasus-init to address this issue:

1. I commented-out the PEGASUS_HOME setting in the generated sites.xml

2. I set default values for OS and ARCH based on the machine where pegaus-init was run. That isn't ideal, but we won't be able to determine the os and arch of the eventual compute site without asking the user, and setting it based on where pegasus-init was run will work better for the tutorials.
